test/Table_Tests/src/Common_Table_Operations/Join/Join_Spec.enso

```diff
@@ -146,9 +146,15 @@ add_join_specs suite_builder setup =
 
             r1 = t1.join t2 join_kind=Join_Kind.Inner
             expect_column_names ["X", "Y", "Z"] r1
-            r1 . at "X" . to_vector . should_equal ["a"]
-            r1 . at "Y" . to_vector . should_equal [1]
-            r1 . at "Z" . to_vector . should_equal [2]
+            case setup.flagged ..Case_Sensitive_Text_Comparison of
+                True ->
+                    r1 . at "X" . to_vector . should_equal ["a"]
+                    r1 . at "Y" . to_vector . should_equal [1]
+                    r1 . at "Z" . to_vector . should_equal [2]
+                False ->
+                    r1 . at "X" . to_vector . should_equal ["a", "a", "B"]
+                    r1 . at "Y" . to_vector . should_equal [1, 1, 2]
+                    r1 . at "Z" . to_vector . should_equal [1, 2, 3]
```

Comment:
with multiple results, let's add `|> materialize |> _.sort ["Z"]` to ensure the ordering is consistent. Adding `materialize` will also mean we execute the query only once, not 3 times (on every `to_vector` call - which is better for high-latency backends such as Snowflake (I'm surprised I didn't notice we don't materialize here when working on Snowflake tests).

Reply:
Done